spec/features/user_creates_bot_spec.rb

```diff
@@ -63,4 +63,23 @@
 
     expect(page).to have_content("You have reached the limits of BAEs")
   end
+
+  scenario "A bot will have a default vocabulary if it was selected" do
+    user = FactoryGirl.create(
+      :user,
+      phone_number: ENV["TWILIO_PHONE_NUMBER"].sub("+1", "")
+    )
+    login(user)
+    visit new_bot_path
+    fill_in "Name", with: "Asia"
+    check('funny_vocabulary')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.